assets/app/app.rb

```diff
@@ -27,8 +27,11 @@ def render
     props = {
       props: { id: 'app' },
       style: {
-        padding: '0.5rem',
+        'background-color': @user&.dig(:settings, :bg_color),
```

Comment:
what happens when this is nil?

Reply:
In theory it should be an "invalid property value" like any other => no background-color set. That’s exactly what happens if no user is logged in.